include/debug/ppi_trace.h

```diff
@@ -45,6 +48,24 @@ void *ppi_trace_config(uint32_t pin, uint32_t evt);
  */
 void *ppi_trace_pair_config(uint32_t pin, uint32_t start_evt, uint32_t stop_evt);
 
+/** @brief Configure a PPI trace pin for tracing a DPPI channel.
+ *
+ * This function allows to trace DPPI triggers without knowing any events being the source of the
+ * trigger. Configuration of events so that they publish to the given DPPI and enabling the DPPI is
+ * out of scope of this function and must be done externally.
+ * This function allows also to trace DPPI channels which are triggered by multiple events or
+ * the set of events publishing to the DPPI channel changes in run-time.
+ *
+ * @note Supported only on platforms equipped with DPPI.
+ *
+ * @param pin		Pin to use for tracing.
+ * @param dppi_ch	DPPI channel number to be traced on the pin.
+ *
+ * @retval true		The configuration succeeded.
+ * @return false	The configuration failed.
+ */
+bool ppi_trace_dppi_config(uint32_t pin, uint32_t dppi_ch);
```

Comment:
`ppi_trace_disable` do not free any resources just disable ppi. For `ppi_trace_dppi_config` we cannot do this way because we could damage functionality of the dppi network the dppi channel is part of. 
I'll rename the function to `ppi_trace_dppi_ch_trace`